components/content-search/index.tsx

```diff
@@ -48,31 +47,87 @@ const StyledSearchControl = styled(SearchControl)`
 	width: 100%;
 `;
 
-const ContentSearch = ({
-	onSelectItem,
-	placeholder,
+interface QueryCache {
```

Comment:
There are many interfaces created for this component, therefore should it be better to move all the interfaces to another file say `definitions.ts`, and just import from there?